Use of unsupported NumPy function 'numpy.gcd' or unsupported use of the function.

```
pytest -sv rbc/tests/test_omnisci_math.py -k test_rational_funcs
```
fails:
```
E                   numba.errors.TypingError: Failed in nopython mode pipeline (step: nopython frontend)
E                   Use of unsupported NumPy function 'numpy.gcd' or unsupported use of the function.
E                   
E                   File "rbc/tests/test_omnisci_math.py", line 280:
E                       def gcd(i, j):
E                           return np.gcd(i, j)
E                           ^
E                   
E                   [1] During: typing of get attribute at /home/pearu/git/xnd-project/rbc/rbc/tests/test_omnisci_math.py (280)
E                   
E                   File "rbc/tests/test_omnisci_math.py", line 280:
E                       def gcd(i, j):
E                           return np.gcd(i, j)
```
Using numba 0.46.

--- a/rbc/tests/test_omnisci_math.py
+++ b/rbc/tests/test_omnisci_math.py
@@ -1,5 +1,6 @@
 import numpy as np
 import pytest
+from rbc.utils import get_version
 rbc_omnisci = pytest.importorskip('rbc.omniscidb')
 
 
@@ -268,6 +269,8 @@ def test_explog_funcs(omnisci):
                 assert(np.isclose(np_fn(x), v))
 
 
+@pytest.mark.skipif(get_version('numba') < (0, 47),
+                    reason="requires numba 0.47 or higher")
 def test_rational_funcs(omnisci):
     omnisci.reset()
 

--- a/rbc/utils.py
+++ b/rbc/utils.py
@@ -7,6 +7,15 @@ import ctypes
 import llvmlite.binding as llvm
 
 
+def get_version(package):
+    """Return a package version as a 3-tuple of integers.
+    """
+    if package == 'numba':
+        import numba
+        return tuple(map(int, numba.__version__.split('.')[:3]))
+    raise NotImplementedError(f'get version of package {package}')
+
+
 def get_local_ip():
     """Return localhost IP.
     """

